### PR TITLE
Add Pi Supply LoRa Gateway HAT

### PR DESCRIPTION
**Pull request by ryanteck**

### CODE REVIEW

**Comment by ryanteck** [comment]
This should fix issue #10

**Comment by ryanteck** [comment]
Will do the changes

File: layers/targets/meta-raspberrypi/recipes-lora/lora-packet-forwarder/lora-packet-forwarder/pislora/global_conf.au915_0.json [hunk 1]
```diff
@@ -0,0 +1,99 @@
+{
```
[the remark this answers is not in the document]
Comment (ryanteck) [reply layers/targets/meta-raspberrypi/recipes-lora/lora-packet-forwarder/lora-packet-forwarder/pislora/global_conf.au915_0.json]: Yeah as I believe we can do the AU915 however I need to do some extra testing first before we do.

File: layers/targets/meta-raspberrypi/recipes-core/gateway-config/gateway-config/gateway-config.sh [hunk 2]
```diff
@@ -12,13 +12,14 @@ do_setup_admin_password() {
...
 do_setup_concentrator_shield() {
-    FUN=$(dialog --title "Setup LoRa concentrator shield" --menu "Select shield:" 15 60 6 \
+    FUN=$(dialog --title "Setup LoRa concentrator shield" --menu "Select shield:" 15 60 7 \
         1 "IMST     - iC880A" \
         2 "IMST     - iC980A" \
         3 "RAK      - RAK2245" \
         4 "RAK      - RAK831" \
         5 "RisingHF - RHF0M301" \
         6 "Sandbox  - LoRaGo PORT" \
+        7 "Pi Supply  - LoRa Gateway HAT" \
```
[the remark this answers is not in the document]
Comment (ryanteck) [reply layers/targets/meta-raspberrypi/recipes-core/gateway-config/gateway-config/gateway-config.sh]: Will do

**Comment by brocaar** [comment]
@ryanteck do you need further help with this?

**Comment by ryanteck** [comment]
Hi Brocarr, Sorry I actually forgot about this. I'll get this changed and fixed today.

**Comment by ryanteck** [comment]
That seems to have done it. We've left the conf for AU in but re-ordered the list now @brocaar 

Let me know if there's any other issues.

**Pull request merged**